src/Raven.Client/Documents/Operations/Revisions/RevertResult.cs

```diff
@@ -216,5 +217,11 @@ public void WarnAboutFailedCollection(string message)
             var id = $"Revert Collection Fail No.{++_failedCollections}";
             Warn(id, message);
         }
+
+        public void WarnAboutFailedDoc(string message)
+        {
+            var id = $"Revert Doc Fail No.{++_failedDocs}";
```

Comment:
Which document is that? What revision? What is the meaning of rhits?

Reply:
this is only the key on the ``Warnong` dictionary,
Look at the `WarnAboutFailedCollection` and `Warn` methods..

Reply:
Reverted